sections/search-page.liquid

```diff
@@ -1,5 +1,4 @@
 {%- if results_count > 0 -%}
```

Comment:
We must be careful here, I think renaming section here might break existing themes.
If someone edited their search template we will have a reference to `sections/search.liquid` which will not be updated when this is deployed and that section will no longer exist.